src/Symfony/Component/Messenger/Command/StopWorkersCommand.php

```diff
@@ -50,7 +50,7 @@ protected function configure(): void
 
 Each worker command will finish the message they are currently processing
 and then exit. Worker commands are *not* automatically restarted: that
-should be handled by something like supervisord.
+should be handled by software like Supervisor or any other process control system.
```

Comment:
I would even simplify to just "should be handled by a process control system."

Reply:
ok lets rewrite like this